pyproject.toml

```diff
@@ -22,6 +22,35 @@ extend_skip_glob = ["**/migrations/*"]
 
 ##############################################
 
+[tool.ruff]
+target-version = "py310"
+line-length = 120
+select = ["F", "E", "B", "W", "N", "UP", "YTT", "FIX", "ASYNC", "A", "DJ", "EXE", "ICN", "G", "SLOT", "TID", "TCH", "INT", "C4", "ISC", "INP", "PIE", "RSE", "RET", "COM","PGH","FLY", "PERF", "PL", "BLE"]  # PYI,Q,SIM,PTH,TRY,RUF
```

Comment:
What are `# PYI,Q,SIM,PTH,TRY,RUF`?

Reply:
We will add an issue to track these rules